src/debug.c

```diff
@@ -34,6 +36,9 @@
 #include <arpa/inet.h>
 #include <signal.h>
 #include <dlfcn.h>
+#include <fcntl.h>
+#include "bio.h"
+#include <unistd.h>
```

Comment:
@oranagra please consider re-order so system headers come first.

Reply:
actually, for some reason in many other C files in redis, the system includes come last.
and in fact, when i include server.h after including signal.h, then server.h complains of unknown type name siginfo_t.
i think this mess should be sorted out some other day.